docs/reference/esql/functions/length.asciidoc

```diff
@@ -1,11 +1,30 @@
 [discrete]
 [[esql-length]]
 === `LENGTH`
-Returns the character length of a string.
+
+*Syntax*
 
 [source,esql]
 ----
-FROM employees
-| KEEP first_name, last_name, height
-| EVAL fn_length = LENGTH(first_name)
+LENGTH(str)
+----
+
+*Parameters*
+
+`str`::
+String expression. If `null`, the function returns `null`.
+
+*Description*
+
+Returns the character length of a string.
+
+*Example*
```

Comment:
There is a separate issue about making sure all functions have a railroad diagram and type table: https://github.com/elastic/elasticsearch/issues/100558 . I wanted to keep that out of scope for this PR, because it is large enough as it is. What do you think?

Reply:
Ah, sorry - of course, let's tackle that in the other issue!